docs/source/index.rst

```diff
@@ -9,8 +9,31 @@ large amounts of simulation data within a simulation workflow.
 
 DPF is an independent, physics-agnostic tool that you can plug into many 
 apps for both data input and data output, including visualization and 
-result plots. It can access data from solver result files and other neutral
-formats, such as CSV, HDF5, and VTK files.
+result plots. By "many", we can draw up an exhaustive list of supported
+apps by DPF and their related format:
+
+.. list-table:: Supported solver and related file format
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Solver
+     - File format
+     - DPF examples
+   * - MAPDL
+     - .rst, .mode, .rfrq, .rdsp
+     - :ref:`_ref_distributed_msup`, :ref:`_ref_average_across_bodies`
+   * - LS DYNA
+     - .d3plot
+     - :ref:`lsdyna_operators`
+   * - Fluent
+     - .cfx, .cas/dat.h5, .res, .flprj
```

Comment:
If it is Fluent, it cannot be cfx. This should rather be:

```
* - Ansys CFF (Common Fluids Format)
   - Fluent CFF restart files: .cas/.dat.h5
   - CFX CFF files: .cas/.dat.cff, 
   - Fluids Project Files: .flprj
```

Reply:
As you can see, I have eliminated `.res` because .res are always supported in beta by CFFSDK, so we should not advertise it